core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -158,8 +158,22 @@ object ConsumerPerformance {
     val elapsedMs: Double = endMs - startMs
     val totalMBRead = (bytesRead * 1.0) / (1024 * 1024)
     val mbRead = ((bytesRead - lastBytesRead) * 1.0) / (1024 * 1024)
+    
```

Comment:
Thank you for your feedback again.

To answer your first comment, There doesn't seem to be any constraint on the value of `reporting-interval`. I can simply pass in the value of `0` (or a negative value) and no validation is performed. So the added checks might prove useful. Unless we validate the value given for `reporting-interval`. I hope I didn't misunderstand your point.

On your second comment, you are right. I'll update the other usage as well.